.travis.yml

```diff
@@ -7,3 +7,6 @@ script:
 node_js:
   - "0.11"
   - "0.10"
+  - "1.0"
+  - "1.1"
+  - iojs
```

Comment:
Rebase on master. This is already there. iojs 1.0 and 1.1 will not be tested, just iojs latest.